ga4gh/client.py

```diff
@@ -15,13 +15,26 @@
 requests.packages.urllib3.disable_warnings()
 
 
-class HTTPClient(object):
-
-    workaroundGoogle = 'google'
-
+class HttpClient(object):
```

Comment:
PEP8 says "Note: When using abbreviations in CapWords, capitalize all the letters of the abbreviation. Thus HTTPServerError is better than HttpServerError."

Might as well stick to PEP8 for this, unless there's a compelling reason.